### PR TITLE
DEV-265: Alt-text for team members profile images

### PR DESCRIPTION
**Pull request by SulthanNK**
- **Ticket No**: DEV-265

- **Work**: 
  - Added alt-text for Junior Developer Group logo in the navigation bar.
  - Added alt-text for team members pictures on the developer page.
  - Added alt-text for team members pictures on the celebration section.

### CODE REVIEW

File: assets/js/celebrations.js [hunk 1]
```diff
@@ -7,55 +7,64 @@ const users = [
...
+        'image': './assets/images/team-members/jay.jpg',
+        'alt': 'Oghenekparobo | gives pose in a white striped red coat and white shirt',
     },
     {
         'fName': 'Debra-Kaye',
         'lName': 'Elliott',
         'celebration': 'Joined Junior Developer Group 🎉',
-        'image': './assets/images/team-members/debra-kaye.jpg'
+        'image': './assets/images/team-members/debra-kaye.jpg',
+        'alt': 'Debra-Kaye | his picture',
```
Comment (J3ffJessie) [inline assets/js/celebrations.js:67]: Debra-Kaye needs to be changed to her picture if possible please.

**Review by J3ffJessie — DISMISSED** [review]
@SulthanNK minor change needed to one of the alt tags if possible.  Overall everything looks in order. Once the change is done I will review and should be able to approve and merge sometime today.

File: assets/js/celebrations.js (hunk 1 above)
Comment (SulthanNK) [reply assets/js/celebrations.js:67]: Looks like Debra-Kaye picture is not in the repository, without a picture the path is alone.

**Comment by SulthanNK** [comment]
@J3ffJessie without the picture, What I need to put as an alt-text

**Comment by J3ffJessie** [comment]
> @J3ffJessie without the picture, What I need to put as an alt-text

Actually for Debra she is an inactive member so she doesn't even need to be listed. We can probably just remove her completely but @mateahoward will have to be the approval on that. If not just change where it says his picture to her picture and it would be fine.

**Comment by SulthanNK** [comment]
Okay, So let's wait until matea checks this issue.

**Comment by mateahoward** [comment]
Awesome job both :D I've added a change to remove a member and will now merge in :D

**Review by mateahoward — APPROVED** [review]

**Review by mateahoward — APPROVED** [review]

**Review by likiipedia — APPROVED** [review]

**Pull request merged**